stdgo/reflect/Reflect.hx

```diff
@@ -29,7 +29,7 @@ enum GT_enum {
 	GT_ptr(elem:GT_enum);
 	GT_slice(elem:GT_enum);
 	GT_array(elem:GT_enum, len:Int);
-	GT_func(input:Array<GT_enum>, output:Array<GT_enum>);
+	GT_func(input:Array<GT_enum>, output:Array<GT_enum>,recv:Array<GT_enum>);
```

Comment:
The spec for the returned Type of  `reflect.Type.Method()` is: https://golang.org/pkg/reflect/#Method

You will see that a comment in the spec specifies the function signature as a `func with receiver as first argument`.

So a separate `recv` argument seems redundant. If it is needed, it should be a single value, not an array.